roles/host-provision/tasks/user-setup.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: user-setup | Create ansible user
+  user:
+    name: "{{ hostprovision_ansible_user }}"
+    uid: "2468"
+    comment: "Ansible toolkit host-provision automation"
+    groups: wheel
```

Comment:
hmmm;  is it safe to assume that this group will exist?  I also believe that, by default, the wheel group requires a (possibly nonexistant) local password to sudo.

An example of a way to maintain such a group;  to be even safer, we may want to consider to create a bms-toolkit specific group to avoid modifying the state of wheel on the control node. 
 https://stackoverflow.com/questions/33359404/ansible-best-practice-for-maintaining-list-of-sudoers

Reply:
No wait, I see you've built logic already to manage sudoers specifically for the user.  So I think the wheel group membership should be entirely unnecessary.

Reply:
Thanks for the catch.
Removing `groups: wheel`